### PR TITLE
Generate PrefixColumnConcatenator with entry point compiler instead of manually.

### PR DESCRIPTION
**Pull request by pieths**

### CODE REVIEW

File: src/python/nimbusml/preprocessing/schema/prefixcolumnconcatenator.py [hunk 1]
```diff
@@ -17,38 +18,15 @@
...
-        The << operator can be used to set this value (see
-        `Column Operator </nimbusml/concepts/columns>`_)
-
-        For example
-         * ColumnConcatenator(columns={'features': ['age', 'parity',
-        'induced']})
-         * ColumnConcatenator() << {'features': ['age', 'parity',
-        'induced']})
-
-        For more details see `Columns </nimbusml/concepts/columns>`_.
```
Comment (ganik) [inline src/python/nimbusml/preprocessing/schema/prefixcolumnconcatenator.py]: we need this doc

File: src/python/nimbusml/internal/core/preprocessing/schema/prefixcolumnconcatenator.py [hunk 2]
```diff
@@ -15,19 +16,12 @@
 from ...base_pipeline_item import BasePipelineItem, DefaultSignature
 
 
-class PrefixColumnConcatenator(BasePipelineItem, DefaultSignature):
+class PrefixColumnConcatenator(
+        BasePipelineItem,
+        DefaultSignature):
     """
 
-    Combines several columns into a single vector-valued column by prefix
```
Comment (ganik) [inline src/python/nimbusml/internal/core/preprocessing/schema/prefixcolumnconcatenator.py]: >Combines several columns into a single vector-valued column by prefix [](start = 4, length = 69)

need this doc

File: src/python/nimbusml/preprocessing/schema/prefixcolumnconcatenator.py (hunk 1 above)
Comment (pieths) [reply src/python/nimbusml/preprocessing/schema/prefixcolumnconcatenator.py]: Fixed in latest commit.

---
In reply to: [346961298](https://github.com/microsoft/NimbusML/pull/364#discussion_r346961298) [](ancestors = 346961298)

File: src/python/nimbusml/internal/core/preprocessing/schema/prefixcolumnconcatenator.py (hunk 2 above)
Comment (pieths) [reply src/python/nimbusml/internal/core/preprocessing/schema/prefixcolumnconcatenator.py]: Fixed in latest commit.

---
In reply to: [346961442](https://github.com/microsoft/NimbusML/pull/364#discussion_r346961442) [](ancestors = 346961442)

File: src/python/nimbusml/internal/core/preprocessing/schema/prefixcolumnconcatenator.py [hunk 3]
```diff
@@ -82,8 +85,7 @@ def _get_node(self, **all_args):
 
         # validate output
         if output_columns is None:
-            raise ValueError(
-                "'None' output passed when it cannot be none.")
```
Comment (ganik) [inline src/python/nimbusml/internal/core/preprocessing/schema/prefixcolumnconcatenator.py]: this one is needed. Pls see example how to use this transform, it should be clear from it

Comment (pieths) [reply src/python/nimbusml/internal/core/preprocessing/schema/prefixcolumnconcatenator.py]: The change has been reverted to the previous code in commit 8afb27968c6eb9e1c3714561dedb64fb9e34e672.

---
In reply to: [347187028](https://github.com/microsoft/NimbusML/pull/364#discussion_r347187028) [](ancestors = 347187028)

**Pull request merged**